bigtop_toolchain/manifests/packages.pp

```diff
@@ -251,6 +254,11 @@
       package { 'ca-certificates':
         ensure => latest
       }
+
+      package { 'devtoolset-9':
+        ensure => installed,
+        require => Package['centos-release-scl']
```

Comment:
We can not accept this PR if you do not care about distributions other than CentOS.

Reply:
GCC version of CentOS7 is too slow and cannot build llvm, so we have to use devtoolset-9, otherwise we need compile GCC from source.
https://github.com/apache/bigtop/pull/968/files#diff-11703cdb6563285ebbc64e4b5e1926b47777dcf7678ba86a2e72a34a481aedcfR25

I'm testing other distros, but I can only cover few of them(maybe centos7 rockylinux8 ubuntu20), it's too many distros here.